sequencer/src/options.rs

```diff
@@ -125,6 +125,20 @@ pub struct Options {
         value_delimiter = ','
     )]
     pub prefunded_builder_accounts: Vec<Address>,
+
+    /// Url we will use for RPC communication with L1. It is
+    /// accpetable to run with out setting this only under `testing`
+    /// feature.
+    #[clap(long, env = "ESPRESSO_SEQUENCER_L1_WS_PROVIDER")]
+    pub l1_provider_url: Option<Url>,
+
+    /// For testing with a pre-merge geth node that does not support
+    /// the finalized block tag we allow setting an environment
+    /// variable to use the latest block instead. This feature is used
+    /// in the OP devnet which uses the docker images built in this
+    /// repo. Therefore it's not hidden behind the testing flag.
+    #[clap(long, env = "ESPRESSO_SEQUENCER_L1_USE_LATEST_BLOCK_TAG")]
```

Comment:
Due to updates in more recent geth versions we may no longer need this variable.